src/appleseed/foundation/meta/tests/test_fp.cpp

```diff
@@ -146,6 +146,44 @@ TEST_SUITE(Foundation_Math_FP_Float)
     {
         EXPECT_EQ(FP<float>::neg_inf(), shift(FP<float>::neg_inf(), -1));
     }
+
+    TEST_CASE(Sign_GivenNegativeAndPositiveFloats_ReturnsOneAndZero)
+    {
+        const float negative = -0.33;
+        const float positive = +0.33;
+        EXPECT_EQ(FP<float>::sign(negative), 1);
```

Comment:
Expected values go first (unlike with Google's unit test framework that doesn't care about the order).